test/unit/sidekiq_mover_test.rb

```diff
@@ -67,14 +67,58 @@ def test_releases_locks_in_source_dc
     assert_equal [1, 2], JobHelper.job_result_queue.map {|arg| arg[0]['num']}
   end
 
+  def test_can_move_retry_queue
+    CrashingJob.perform_async(num: 1)
+
+    begin
+      work_off_jobs(@source)
+    rescue CrashingJob::Error
+    end
+
+    assert_equal 1, @source.zcard('retry')
+
+    sidekiq_delayed_mover('retry')
+    assert_equal 1, @dest.zcard('retry')
```

Comment:
if the item is not removed from `@source` can we assert that the number in `@dest` == number in `@source`

Reply:
It should be removed from the source though.

Reply:
If that's the case we can asset that source is 0 and dest has the right number